_pa_run(u'/home/Cryzzy/mysite/paperhistoryOpenAI/main.py')
Traceback (most recent call last):
  File "/home/Cryzzy/mysite/paperhistoryOpenAI/main.py", line 4, in <module>
    from flask_mysqldb import MySQL
ModuleNotFoundError: No module named 'flask_mysqldb'
>>> pip3 install --user flask_mysqldb
clear
pip3 install --user flask_mysqldb
exit()
